### DISCUSSION TITLE
bloom filter sharp rectangular cut in svg

### DISCUSSION THREAD

**Discussion by thewayur** [pixijs/pixijs#7371] [Q&A] [answered]
how to smooth / give more area to these sharp  rectangular cut?

![image](https://user-images.githubusercontent.com/2944767/113423078-7d141d80-93eb-11eb-9201-66d64933ef62.png)

**Comment by bigtimebuddy** [comment, marked as answer]
Probably add `padding` to the filter.

**Reply by thewayur** [reply]
ok, it worked, for future people(including me) here is a simple fix:

`

const **bloomFltr** = new PIXI.filters.BloomFilter();
const heart = new PIXI.Sprite(heartTexture);
heart.filters = [**bloomFltr**,blurFltr]
**bloomFltr.blur =120;**
**_bloomFltr.padding =5;_**

`
